poetry.lock

```diff
@@ -645,16 +652,18 @@ version = "0.6.0"
 more-itertools = "*"
 
 [extras]
+datetime = ["aniso8601"]
```

Comment:
maybe we should add this as default dependency :)

Reply:
I was thinking that someone who doesn't need the DateTime type, might not want to install this dependency. But maybe it is common enough to warrant making this a hard dependency.

Reply:
@jgosmann I'd add this as default, graphene does that too (with another library) :)

Reply:
Did that.